command/agent/agent_endpoint.go

```diff
@@ -393,6 +396,61 @@ func (s *HTTPServer) AgentNodeMaintenance(resp http.ResponseWriter, req *http.Re
 	return nil, nil
 }
 
+func (s *HTTPServer) AgentMonitor(resp http.ResponseWriter, req *http.Request) (interface{}, error) {
+	// Only GET supported
+	if req.Method != "GET" {
+		resp.WriteHeader(405)
+		return nil, nil
+	}
+
```

Comment:
We are going to add an ACL here, but I feel like this is a bit of a security vuln until we do that.  Just so we don't expose anything weird while that work is going on in master, lets take a token here and call the Raft endpoint:

https://github.com/hashicorp/consul/blob/master/command/agent/operator_endpoint.go#L23-L26

This'll vet that they have operator read privs if it doesn't return an error (just throw away the response), which protects this with something while we develop ACLs.